config.go

```diff
@@ -43,7 +43,10 @@ type FileInfo struct {
 //  ./config/${ENVIRONMENT}.yaml
 //  ./config/secrets.yaml
 // The provider is also amended with a command line provider that reads
-// a roles flag.
+// a roles flag. It can be skipped by seting commandline value to false
+// at the root level of config file:
```

Comment:
we don't refer to the `root level of config file` anywhere else. Just call out `base.yaml`